file_organisation.qmd

```diff
@@ -90,23 +90,27 @@ config
 
 ### `metadata.yml`
 
-XXX
+The file `metadata.yml` documents dataset-level metadata, including a database description, authors, and funders.
 
 ### `traits.yml`
 
 The file `traits.yml` provides the trait definitions used to compile AusTraits, including allowable trait values. The [trait definitions](trait_definitions.html) are fully described in an additional vignette. A `.yml` file is a structured data file where information is presented in a hierarchical format (see [appendix for details](yaml.html)).
 
 ### `taxon_list.csv`
 
-The file `taxon_list.csv` is our master list of known taxa. 
+The file `taxon_list.csv` is our master list of taxa in the trait database. 
 
-XXX Explan this more. Only show essential variables in table below.
+It includes all unique taxon names after typos have been corrected (through taxonomic_updates). It includes both accepted/valid taxon concepts and outdated taxonomic names. It includes taxon names indicating a taxon that can be identified to species and names that indicate name-alignment to a lower taxon rank.
 
-<!-- Each species is listed once, with links to species' identifiers provided by the Australian Plant Name Index (APNI). The file `taxon_list.csv` is added to if a study includes taxa not previously represented in AusTraits. These can be names included in either the APC/APNI, compilations of taxonomic concepts (APC) or names (APNI) for plants that are either native to or naturalised in Australia, or taxa without recognised names. -->
+For any species (& infraspecific taxon concepts) or genera that align with known taxon concepts, identifiers for these taxon concepts are included if available in a master taxon list. For instance for AusTraits, identifiers from the two vascular plants National Species Lists (NSL), the APC (Australian Plant Census) and the Australian Plant Name Index (APNI) are included.
+
+In the file, `cleaned_name` refers to the taxon name after any typos have been corrected, while `taxon_name` is the taxon name following updates to the currently accepted/valid taxon name (when available).
+
+<!-- Each species is listed once, with links to species' identifiers provided by the APNI. The file `taxon_list.csv` is added to if a study includes taxa not previously represented in AusTraits. These can be names included in either the APC/APNI, compilations of taxonomic concepts (APC) or names (APNI) for plants that are either native to or naturalised in Australia, or taxa without recognised names. -->
 
 ```{r, results='show'}
-read_csv("data/taxon_list.csv", show_col_types = FALSE) %>%
-  select(taxon_name, family, scientific_name_authorship, taxonomic_reference, cleaned_name_taxonomic_status, cleaned_scientific_name_id
+read_csv("config/taxon_list.csv", show_col_types = FALSE) %>%
```

Comment:
In this repo it's in data, not config

Reply:
@dfalster I'm confused. In traits.build it isn't included at all. In traits.build-template and austraits.build it is in config. I thought this `file_organisation.qmd` was describing the file structure of a traits.build database, not the traits.build repo.